examples/amm/artifacts/application.json

```diff
@@ -46,7 +46,7 @@
9nbG9iYWxfZ2V0Cj09Ci8vIGFzc2V0IGEgaW5jb3JyZWN0CmFzc2VydApsb2FkIDc2CnR4bmFzIEFzc2V0cwpieXRlY18xIC8vICJiIgphcHBfZ2xvYmFsX2dldAo9PQovLyBhc3NldCBiIGluY29ycmVjdAphc3NlcnQKbG9hZCA3NApndHhucyBYZmVyQXNzZXQKYnl0ZWNfMCAvLyAiYSIKYXBwX2dsb2JhbF9nZXQKPT0KbG9hZCA3NApndHhucyBYZmVyQXNzZXQKYnl0ZWNfMSAvLyAiYiIKYXBwX2dsb2JhbF9nZXQKPT0KfHwKLy8gYXNzZXQgYSBvciBhc3NldCBiIGluY29ycmVjdAphc3NlcnQKbG9hZCA3NApndHhucyBBc3NldEFtb3VudAppbnRjXzAgLy8gMAo+Ci8vIGFtb3VudCBtaW5pbXVtIG5vdCBtZXQKYXNzZXJ0CmxvYWQgNzQKZ3R4bnMgU2VuZGVyCnR4biBTZW5kZXIKPT0KLy8gaW52YWxpZCBzZW5kZXIKYXNzZXJ0Cmdsb2JhbCBDdXJyZW50QXBwbGljYXRpb25BZGRyZXNzCmxvYWQgNzQKZ3R4bnMgWGZlckFzc2V0CmFzc2V0X2hvbGRpbmdfZ2V0IEFzc2V0QmFsYW5jZQpzdG9yZSA3OApzdG9yZSA3NwpnbG9iYWwgQ3VycmVudEFwcGxpY2F0aW9uQWRkcmVzcwpsb2FkIDc0Cmd0eG5zIFhmZXJBc3NldApieXRlY18wIC8vICJhIgphcHBfZ2xvYmFsX2dldAo9PQpibnogc3dhcF8xMV9sNQpieXRlY18wIC8vICJhIgphcHBfZ2xvYmFsX2dldApzd2FwXzExX2wyOgphc3NldF9ob2xkaW5nX2dldCBBc3NldEJhbGFuY2UKc3RvcmUgODAKc3RvcmUgNzkKbG9hZCA3OAovLyBtaXNzaW5nIHJlcXVpcmVkIGJhbGFuY2VzCmFzc2VydApsb2FkIDgwCi8vIG1pc3NpbmcgcmVxdWlyZWQgYmFsYW5jZXMKYXNzZXJ0CmxvYWQgNzQKZ3R4bnMgQXNzZXRBbW91bnQKbG9hZCA3Nwpsb2FkIDc0Cmd0eG5zIEFzc2V0QW1vdW50Ci0KbG9hZCA3OQpjYWxsc3ViIHRva2Vuc3Rvc3dhcF8xNQpzdG9yZSA4MQpsb2FkIDgxCmludGNfMCAvLyAwCj4KLy8gb3V0Z29pbmcgYW1vdW50IHRvbyBsb3cKYXNzZXJ0CnR4biBTZW5kZXIKbG9hZCA3NApndHhucyBYZmVyQXNzZXQKYnl0ZWNfMCAvLyAiYSIKYXBwX2dsb2JhbF9nZXQKPT0KYm56IHN3YXBfMTFfbDQKYnl0ZWNfMCAvLyAiYSIKYXBwX2dsb2JhbF9nZXQKYiBzd2FwXzExX2w2CnN3YXBfMTFfbDQ6CmJ5dGVjXzEgLy8gImIiCmFwcF9nbG9iYWxfZ2V0CmIgc3dhcF8xMV9sNgpzd2FwXzExX2w1OgpieXRlY18xIC8vICJiIgphcHBfZ2xvYmFsX2dldApiIHN3YXBfMTFfbDIKc3dhcF8xMV9sNjoKbG9hZCA4MQpjYWxsc3ViIGRvYXhmZXJfNgpieXRlYyA0IC8vICJyIgpjYWxsc3ViIGNvbXB1dGVyYXRpb181CmFwcF9nbG9iYWxfcHV0CnJldHN1YgoKLy8gdG9rZW5zX3RvX2J1cm4KdG9rZW5zdG9idXJuXzEyOgpzdG9yZSA1MwpzdG9yZSA1MgpzdG9yZSA1MQpsb2FkIDUyCmxvYWQgNTMKbXVsdwppbnRjXzAgLy8gMApsb2FkIDUxCmRpdm1vZHcKcG9wCnBvcApzd2FwCiEKYXNzZXJ0CnJldHN1YgoKLy8gdG9rZW5zX3RvX21pbnQKdG9rZW5zdG9taW50XzEzOgpzdG9yZSA3MApzdG9yZSA2OQpzdG9yZSA2OApzdG9yZSA2NwpzdG9yZSA2Ngpsb2FkIDY5CmludGNfMiAvLyAxMDAwCm11bHcKaW50Y18wIC8vIDAKbG9hZCA2NwpkaXZtb2R3CnBvcApwb3AKc3dhcAohCmFzc2VydApzdG9yZSA3MQpsb2FkIDcwCmludGNfMiAvLyAxMDAwCm11bHcKaW50Y18wIC8vIDAKbG9hZCA2OApkaXZtb2R3CnBvcApwb3AKc3dhcAohCmFzc2VydApzdG9yZSA3Mgpsb2FkIDcxCmxvYWQgNzIKPApibnogdG9rZW5zdG9taW50XzEzX2wyCmxvYWQgNzIKYiB0b2tlbnN0b21pbnRfMTNfbDMKdG9rZW5zdG9taW50XzEzX2wyOgpsb2FkIDcxCnRva2Vuc3RvbWludF8xM19sMzoKbG9hZCA2NgptdWx3CmludGNfMCAvLyAwCmludGNfMiAvLyAxMDAwCmRpdm1vZHcKcG9wCnBvcApzd2FwCiEKYXNzZXJ0CnJldHN1YgoKLy8gdG9rZW5zX3RvX21pbnRfaW5pdGlhbAp0b2tlbnN0b21pbnRpbml0aWFsXzE0OgoqCnNxcnQKaW50Y18yIC8vIDEwMDAKLQpyZXRzdWIKCi8vIHRva2Vuc190b19zd2FwCnRva2Vuc3Rvc3dhcF8xNToKc3RvcmUgODQKc3RvcmUgODMKc3RvcmUgODIKbG9hZCA4MgppbnRjXzIgLy8gMTAwMApwdXNoaW50IDUgLy8gNQotCm11bHcKbG9hZCA4NAp1bmNvdmVyIDIKZGlnIDEKKgpjb3ZlciAyCm11bHcKY292ZXIgMgorCnN3YXAKaW50Y18wIC8vIDAKbG9hZCA4MwppbnRjXzIgLy8gMTAwMAoqCmxvYWQgODIKaW50Y18yIC8vIDEwMDAKcHVzaGludCA1IC8vIDUKLQoqCisKZGl2bW9kdwpwb3AKcG9wCnN3YXAKIQphc3NlcnQKcmV0c3Vi",
```

Comment:
Reviewer note - The highlighted diff is unrelated to the `pragma` change.  It comes from other stale generated artifact changes in the upstream PR.